x/subscription/keeper/subscription_test.go

```diff
@@ -317,6 +320,49 @@ func TestSubscriptionAdminProject(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestMonthlyRechargeCU(t *testing.T) {
```

Comment:
in the tests we want to use the message server (ts.servers) to simulate the whole process and not call the keepers functions

Reply:
Unsure what you propose.
Monthly recharge is an internal activity that is carried out by the code periodically and not as a result of a message to a server. Specifically, the subscription code checks for subscription whose current month just expires, and if so runs the needed logic.